src/tests/ftest/util/server_utils_params.py

```diff
@@ -117,9 +116,6 @@ def __init__(self, filename, common_yaml):
         self.control_log_file = LogParameter(log_dir, None, "daos_control.log")
         self.helper_log_file = LogParameter(log_dir, None, "daos_admin.log")
         self.telemetry_port = BasicParameter(None, 9191)
-        default_enable_vmd_val = os.environ.get("DAOS_ENABLE_VMD", "False")
-        default_enable_vmd = ast.literal_eval(default_enable_vmd_val)
-        self.enable_vmd = BasicParameter(None, default_enable_vmd)
```

Comment:
This code is also required for CI.  This class provides the ability for the test code to manipulate any server config file parameter so at the very least it should include `enable_vmd`/`disable_vmd` to allow us to write tests to verify the parameter.

Reply:
So after getting clarification I would just update this comment to provide the ability to set the value, but not include it in the server config by default with:
```
self.disable_vmd = BasicParameter(None)
```